### ISSUE TITLE
Error Installing on Raspberry

### ISSUE THREAD

**Issue by NDNELSON** [atom/node-keytar#91]
I installed keytar in a raspberry pi 3 with jessie.
Firstly I install this Debian/Ubuntu: sudo apt-get install libsecret-1-dev
Secondly I did this:    git clone https:github.com/atom/node-keytar
Finally I use:  npm install keytar

Everything seems fine . I am using keytar for hap-client plugin and when I run my node code I get this error:

Error: Cannot find module '../build/Release/keytar.node
I don't know why keytar.node is not in the directory.

I will apreciate your support a lot

**Comment by surferandi** [comment]
I run into the same problems as @NDNELSON. I also want to use the hap-client (within the iobroker project on a raspberry pi 3).

It seems that the file keytar.node which is required within lib/keytar.js is not build, instead just keytar.a is copied to the release folder:

```
pi@raspi:/opt/iobroker/node_modules/iobroker.javascript $ sudo npm install keytar@4.1.0

> keytar@4.1.0 install /opt/iobroker/node_modules/iobroker.javascript/node_modules/keytar
> node-gyp rebuild

make: Verzeichnis „/opt/iobroker/node_modules/iobroker.javascript/node_modules/keytar/build“ wird betreten
  CXX(target) Release/obj.target/keytar/src/async.o
  CXX(target) Release/obj.target/keytar/src/main.o
  CXX(target) Release/obj.target/keytar/src/keytar_posix.o
  AR(target) Release/obj.target/keytar.a
  COPY Release/keytar.a
make: Verzeichnis „/opt/iobroker/node_modules/iobroker.javascript/node_modules/keytar/build“ wird verlassen
iobroker.javascript@3.4.0 /opt/iobroker/node_modules/iobroker.javascript
└── (empty)

npm ERR! code 1
```

Any help is appreciated! Thanks!

**Comment by surferandi** [comment]
Submitted PR https://github.com/atom/node-keytar/pull/92 to fix this. With this change a keytar.node file is built.

@NDNELSON The next problem I run into is that keytar relies on dbus messaging and I did not yet manage to solve this. If you get it running please tell how ;)

**Comment by surferandi** [comment]
@NDNELSON Either wait for the PR to be merged into a new version or directly reinstall from the fork: `npm install git+https://github.com/surferandi/node-keytar

**Comment by shiftkey** [comment]
I've published [`v4.9.0`](https://github.com/atom/node-keytar/releases/tag/v4.9.0) with the proposed changed from #92 - let me know if there's any other feedback you have around this area.
